### PR TITLE
Admin may manage Client, Workspace, and People data

### PR DESCRIPTION
**Pull request by zspencer**
See: https://github.com/zinc-collective/convene/issues/103

This is a very rough draft, and is only intended to give us a baseline
to start with so I stop going into heroku rails console to manage rooms
and such.

- [x] Make sure we explode horribly if ENV[‘ADMIN_USERNAME’] and ENV[‘ADMIN_PASSWORD’] are not set.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '3.1'
+
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: password
+    ports:
+      - 5432:5432
```
[the remark this answers is not in the document]
Comment (zspencer) [reply docker-compose.yml:10]: Accidentally committed it. One of my computers does not have postgresql-server installed; so the `docker-compose.yml` file allows me to spin up the service-level dependencies without installing postgresql. 

Probably needs it's own PR with documentation updates as well.

File: convene-web/app/controllers/admin/application_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# All Administrate controllers inherit from this
+# `Administrate::ApplicationController`, making it the ideal place to put
+# authentication logic or other before_actions.
+#
+# If you want to add pagination or other controller-level concerns,
+# you're free to overwrite the RESTful controller actions.
+module Admin
+  class ApplicationController < Administrate::ApplicationController
+    extend Configuration::Configurable
```
[the remark this answers is not in the document]
Comment (zspencer) [reply convene-web/app/controllers/admin/application_controller.rb:9]: Thank you! I was doing this on my ipad, which does not (yet) let me view the Heroku logs so I couldn't figure out why the build was failing!

**Comment by user512** [comment]
Looks great and tested on review app.
Able to create resources but likely need more config changes to make it play nice with uuid.

**Comment by zspencer** [comment]
Yeaaaaaa, turns out I had overlooked some of the stuff regarding linking. I've fixed that, and also did some cleanup of the dashboards. Thanks!

**Pull request merged**